packages/config/src/projects/other/da-beat/types/DaBridge.ts

```diff
@@ -115,9 +116,9 @@ type CommonDaBridge = {
   }
 
   /**
-   * List of projects given bridge is being used in
+   * List of projects' references given bridge is being used in
    */
-  usedIn: ProjectId[]
+  usedIn: DaProjectReference[]
```

Comment:
I kind a don't like the word reference here, especially that usedIn is the key.

Reply:
https://github.com/l2beat/l2beat/pull/4520/commits/5da67959e387e859cea288e24e632a2f4d9c1dca